frontend/lib/strapi-sdk/generated/sdk.ts

```diff
@@ -1,1357 +1,2347 @@
 export type Maybe<T> = T | null;
-export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
-export type MakeOptional<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]?: Maybe<T[SubKey]> };
-export type MakeMaybe<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]: Maybe<T[SubKey]> };
+export type Exact<T extends { [key: string]: unknown }> = {
```

Comment:
What's up with this strapi diff?

Reply:
This is an auto generated file... Changes are due to formatting. It's weird that it was different before, but it should be correct now